### ISSUE TITLE
Alert : Undefined is not an object

### ISSUE THREAD

**Issue by yshwanth** [jensgertsen/sparkkiosk#3]
![bluewallet-spark](https://user-images.githubusercontent.com/42468133/162417145-2296d1f7-5c74-4a2e-822b-281022f97dc0.jpeg)

**Comment by johnpc** [comment]
I am experiencing a similar error on blue wallet on iOS 
![IMG_0215](https://user-images.githubusercontent.com/4824042/174702965-ba36de98-1531-43e9-8bd1-d63245b5b0b3.jpeg). 

I generated the LNURL on Sparkkiosk on umbrel. I can't seem to receive payment from it though. This is the QR code that I'd like to post up:

![One_Dollar](https://user-images.githubusercontent.com/4824042/174703186-8aa97746-c783-4244-a97a-5c39477cb44b.png)

**Comment by jensgertsen** [comment]
Hmm, sorry for late reply, the encoded url (27ex5ypixioc4qxhmozkmgwvjmixz5rtiekwrlgrgo6ookwqaff3jmid.onion/lnurl/26091efbfa68707cb7123760cab3c9b0) looks ok, but apperently the service is no longer available (sparkkiosk needs to be running as a service). Feel free to post a new qr, I'll take a look.

**Comment by johnpc** [comment]
I am using spark kiosk via umbrel - is there any way I can ensure the service is running? The app is certainly running, I can access the web ui on port `21214` of my device. I've just now generated a new QR, and I see the same error when attempting to send money to it on blue wallet. Slightly different this time actually, error was `lnurl-pay expected, found tag undefined`

![Testing_for_GitHub](https://user-images.githubusercontent.com/4824042/174930502-3de18301-8530-41bc-b96e-b92ff7831167.png)

It looks like the server is running on docker on my Umbrel, unless I'm misunderstanding something:

```
umbrel@umbrel:~/umbrel $ docker ps | grep sparkkiosk
f5f09e5c00f1   getumbrel/app-proxy:v0.1.0             "docker-entrypoint.s…"   13 days ago    Up 4 days                       0.0.0.0:21214->21214/tcp, :::21214->21214/tcp                                                                                         sparkkiosk_app_proxy_1
980aff6479fe   jensgertsen/sparkkiosk:v1.0.0          "docker-entrypoint.s…"   13 days ago    Up 4 days                       21214/tcp                                                                                                                             sparkkiosk_web_1
cd1162bbf0e7   lncm/tor:0.4.7.7                       "tor"                    13 days ago    Up 4 days                       9050-9051/tcp, 29050-29051/tcp                                                                                                        sparkkiosk_tor_server_1
```

**Comment by jensgertsen** [comment]
Ok - The payload in that qr is: (you can decode lnurls at:https://lnurl.fiatjaf.com/codec/)
http://27ex5ypixioc4qxhmozkmgwvjmixz5rtiekwrlgrgo6ookwqaff3jmid.onion/lnurl/a706f8255376c811ca62e87ed421965e - when accessing this (via tor), an Umbrel login screen appears instead of expected invoice json
![Screenshot 2022-06-22 at 08 38 46](https://user-images.githubusercontent.com/8363477/174960879-3d054c6a-240f-4aa3-81c6-fa7377e3f0d6.png).
I haven't tested in umbrel 0.5, maybe something changed to "allow remote access" - will investigate.

**Comment by jensgertsen** [comment]
Yup, broken in v0.5. The fix should be pretty easy, but I cant upgrade to 0.5 atm to test (upgrade fails). Will fix asap.

**Comment by johnpc** [comment]
That makes a lot of sense - let me know if there's anything I can do to help. And thanks a lot for building spark kiosk - it's the easiest way I've found to start accepting lightning payments (assuming it works of course)!

**Comment by jensgertsen** [comment]
Thanks - the fix should be live now, so hopefully tha app works again (unfortunately the tor network is apparently having issues)
